session_0/fizz_buzz.rb

```diff
@@ -36,6 +36,22 @@
 #
 # We will raise an `ArgumentError` exception to let the caller know that
 # their function arguments were incorrect.
-def fizz_buzz(n:, x: 3, y: 5)
+def fizz_buzz(n:, x:3, y:5)
+  if(x<=0 or y<=0 or n<0)
+    raise ArgumentError.new("Function arguments are incorrect") #divisibility by 0 or -ve number is not possible
+  end
+  result = []
+  n.times do |time|
```

Comment:
See if `map` function is useful here: https://www.rubyguides.com/2018/10/ruby-map-method/